src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitSessionResource.kt

```diff
@@ -45,8 +46,21 @@ class VisitSessionResource(
     ]
   )
   fun getVisitSessions(
-    @Schema(description = "NOMIS Prison Identifier", example = "MDI", required = true)
-    @PathVariable prisonId: String
+    @RequestParam(value = "prisonId", required = true)
+    @Parameter(
+      description = "Query by NOMIS Prison Identifier",
+      example = "MDI"
+    ) prisonId: String,
+    @RequestParam(value = "min", required = false)
+    @Parameter(
+      description = "Minimum number of days notice from the current date",
+      example = "2"
+    ) min: Long?,
+    @RequestParam(value = "max", required = false)
+    @Parameter(
+      description = "Maximum number of days to book-ahead from the current date",
```

Comment:
meh..worth mentioning that this defaults if not provided?

Reply:
Good point, ill update.